src/transforms/rotations.jl

```diff
@@ -0,0 +1,82 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    Rotation(rotation)
+
+Perform the rotation represented by `rotation`, a
+`ReferenceFrameRotation` object from the `ReferenceFrameRotations` 
+library.
+
+## Examples
+
+```julia
+using ReferenceFrameRotations
+Rotation(ReferenceFrameRotations.EulerAngleAxis(pi/4, [1, 0, 0]))
```

Comment:
We don't need to import the package in the examples. Also we should probably reexport ReferenceFrameRotations.jl if it is not exported already.